pkg/testcommon/testcommon.go

```diff
@@ -200,3 +199,19 @@ func ClearDockerEnv() {
 		}
 	}
 }
+
+// checkErr exits with a log.Fatal() if an error is encountered.
+// It is normally used for errors that we never expect to happen, and don't have any normal handling technique.
+// From https://davidnix.io/post/error-handling-in-go/
```

Comment:
Really hope we can get by with not copying this, again the black box testing with different _test package may help.